README.md

```diff
@@ -125,6 +125,8 @@ jobs:
     permissions:
       id-token: write # This is required for requesting the JWT
       contents: read  # Required by actions/checkout@v3 to checkout a private repository
+      pull-requests: write # for dorny/paths-filter to read pull requests and endorctl to write pr comments
```

Comment:
Could we remove the dorny/paths-filter part since we are not using this action here?